src/test/kotlin/com/charleskorn/kaml/YamlReadingTest.kt

```diff
@@ -831,6 +838,119 @@ object YamlReadingTest : Spek({
                 }
             }
 
+            val sealedYaml = Yaml(context = sealedModule)
+
+            context("given some tagged input representing an object where the resulting type should be a sealed class (int)") {
+                val input = """
+                    element: !<sealedInt>
```

Comment:
Is there any way to support `!sealedInt` instead of `!<sealedInt>`?

Reply:
I think that can be done via the `%TAG` directive, but I'm not sure how to do that.